docs/topics/snippets.md

```diff
@@ -298,10 +298,11 @@ class Advert(RevisionMixin, models.Model):
 
     @property
     def revisions(self):
+        # Some custom logic here if necessary
         return self._revisions
 ```
 
-If your snippet model defines relations using Django's {class}`~django.db.models.ForeignKey` or {class}`~django.db.models.ManyToManyField`, you need to change the model class to inherit from `modelcluster.models.ClusterableModel` instead of `django.models.Model` and replace the `ForeignKey` and `ManyToManyField` with `ParentalKey` and `ParentalManyToManyField`, respectively. This is necessary in order to allow the relations to be stored in the revisions. For example:
+If your snippet model defines relations using Django's {class}`~django.db.models.ManyToManyField`, you need to change the model class to inherit from `modelcluster.models.ClusterableModel` instead of `django.models.Model` and replace the `ManyToManyField` with `ParentalManyToManyField`. For inlined models, you also need to use `ParentalKey` instead of `ForeignKey`. This is necessary in order to allow the relations to be stored in the revisions. See the [](tutorial_categories) section of the tutorial for more details. For example:
```

Comment:
Re "For inlined models, you also need to use `ParentalKey` instead of `ForeignKey`." - if I'm not mistaken, this is always the case for anything using an InlinePanel, regardless of whether the model is revisioned or not. I can't decide whether we're better off leaving this sentence as is, explicitly acknowledging it by saying something like "Inline models should continue to use `ParentalKey`, just as they do for non-revision-enabled snippets.", or just leaving it out entirely.

Reply:
Ah yep, I thought about it too when I wrote it but I wasn't sure. I think we could acknowledge it explicitly for completeness' sake. Merging now, thanks!